tests/smoke/install/task.yaml

```diff
@@ -1,8 +1,8 @@
 summary: Check that installing and running a snap works
 
 restore: |
-    rm -f stderr.log
-    rm -f stdout.log
+    rm -f /home/test/stderr.log
```

Comment:
I think files at both locations get created, so cleanup should happen in $PWD and in /home/test

Reply:
The dir $PWD is automatically cleaned up during restore.